website/src/css/custom.css

```diff
@@ -31,3 +31,9 @@ html[data-theme='dark'] {
     --ifm-font-size-base: 17px;
   }
 }
+
+.menu__list .menu__list .menu__list .menu__link {
+  font-size: 0.75em;
+  font-weight: normal;
+  color: #999;
```

Comment:
We should use Infima variables. Also this CSS should be in the docs component and not just for our website.

Reply:
i wasn’t quite sure how much we wanted this in common, and since we plan to rewrite the sidebar so i changed it in the website only